Out of support (#32877)
dotnet/AspNetCore.Docs: aspnetcore/includes/out-of-support.md

> > [!WARNING] This version of ASP.NET Core is no longer supported. For more information, see [.NET and .NET Core Support Policy](https://dotnet.microsoft.com/platform/support/policy/dotnet-core). For the current release, see the [.NET 8 version of this article](?view=aspnetcore-8.0&preserve-view=true).

Author: ```suggestion
> [!WARNING]
> This version of ASP.NET Core is no longer supported. For more information, see [.NET and .NET Core Support Policy](https://dotnet.microsoft.com/platform/support/policy/dotnet-core). For the current release, see the [.NET 8 version of this article](?view=aspnetcore-8.0&preserve-view=true).
```

> This version of ASP.NET Core is no longer supported. Use the version selector in the documentation to select a supported version. For more information, see the [.NET and .NET Core Support Policy](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) page.
> :::image type="content" source="https://github.com/dotnet/AspNetCore.Docs/assets/3605364/9145fe9d-017e-40fd-b6b9-7bccd1bd2325" alt-text="version selector":::

Reviewer: Suggest we send them to the current version like we do with not-latest and omit the image of the version selector.
```suggestion
This version of ASP.NET Core is no longer supported. For more information, see [.NET and .NET Core Support Policy](https://dotnet.microsoft.com/platform/support/policy/dotnet-core). For the current release, see the [.NET 8 version of this article](?view=aspnetcore-8.0&preserve-view=true).
```

This version of ASP.NET Core is no longer supported. Use the version selector in the documentation to select a supported version. For more information, see the [.NET and .NET Core Support Policy](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) page.
:::image type="content" source="https://github.com/dotnet/AspNetCore.Docs/assets/3605364/9145fe9d-017e-40fd-b6b9-7bccd1bd2325" alt-text="version selector":::

> ![image](https://github.com/dotnet/AspNetCore.Docs/assets/3605364/9145fe9d-017e-40fd-b6b9-7bccd1bd2325)

Author: ```suggestion
```
